contracts/contracts/bridge/IBridge.sol

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: Apache 2
```

Comment:
comment around the folder.
`/contracts/contracts`

can it better be: `/contracts/src` ?

and then we'd also have:
`/contracts/bin/generated`
etc

Reply:
/bin/generated? Is this for the go generated files or the build artifacts?  (or just a typo?)